x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/aggregation/AllFirstBytesRefByTimestampAggregator.java

```diff
@@ -27,121 +28,145 @@
 import org.elasticsearch.compute.data.LongBlock;
 import org.elasticsearch.compute.operator.DriverContext;
 import org.elasticsearch.core.Releasables;
+
+import java.util.BitSet;
 // end generated imports
 
 /**
  * A time-series aggregation function that collects the First occurrence value of a time series in a specified interval.
- * This class is generated. Edit `X-AllValueByTimestampAggregator.java.st` instead.
+ * This class is generated. Edit `X-AllValueByTimestafmpAggregator.java.st` instead.
  */
 @Aggregator(
     {
-        @IntermediateState(name = "timestamps", type = "LONG"),
-        @IntermediateState(name = "values", type = "BYTES_REF"),
-        @IntermediateState(name = "seen", type = "BOOLEAN"),
-        @IntermediateState(name = "hasValue", type = "BOOLEAN") }
+        @IntermediateState(name = "observed", type = "BOOLEAN"),
```

Comment:
Is this "observed" the same as "seen"? If that's it, I would reuse "seen", as we use it everywhere else